h5py/_hl/group.py

```diff
@@ -113,13 +119,19 @@ def create_dataset(self, name, shape=None, dtype=None, data=None, **kwds):
             (Scalar) Use this value for uninitialized parts of the dataset.
         track_times
             (T/F) Enable dataset creation timestamps.
+        track_order
+            (T/F) Track attribute creation order if True. If omitted use
+            global default h5.get_config().track_order.
         external
             (List of tuples) Sets the external storage property, thus
             designating that the dataset will be stored in one or more
             non-HDF5 file(s) external to the HDF5 file. Adds each listed
             tuple of (file[, offset[, size]]) to the dataset's list of
             external files.
         """
+        if 'track_order' not in kwds:
```

Comment:
You can write this as `kwds.setdefault('track_order', h5.get_config().track_order)`.   

I am 50/50 on this change though.  On one hand, it get rid of a conditional, but it force accessing `h5p.get_config()`.